neqo-transport/tests/common/mod.rs

```diff
@@ -48,7 +48,6 @@ pub fn connected_server(server: &mut Server) -> ActiveConnectionRef {
         .iter()
         .filter(|c: &&ActiveConnectionRef| *c.borrow().state() == State::Confirmed);
     let c = confirmed.next().expect("one confirmed");
-    assert!(confirmed.next().is_none(), "only one confirmed");
```

Comment:
The unit test `different_initials_same_path` has two clients connect to the same server using this helper function (i.e. `connected_server`). The second connection attempt will fail, as this helper function asserts a single confirmed connection only.